packages/cohort/src/main.rs

```diff
@@ -1,22 +1,29 @@
 // $coverage:ignore-start
+use std::sync::Arc;
 use tokio::signal;
 
 use cohort::config_loader::ConfigLoader;
 use cohort::core::Cohort;
+use cohort::model::bank_account::BankAccount;
+use cohort::state::model::Snapshot;
+use cohort::state::postgres::data_store::DataStore;
+use cohort::state::postgres::database::Database;
 
 #[tokio::main]
 async fn main() -> Result<(), String> {
     env_logger::builder().format_timestamp_millis().init();
 
-    let (cfg_agent, cfg_kafka) = ConfigLoader::load()?;
+    let (cfg_agent, cfg_kafka, cfg_db) = ConfigLoader::load()?;
     tokio::spawn(async move {
-        let agent = Cohort::make_agent(cfg_agent, cfg_kafka).await;
-        let mut cohort = Cohort::new(agent);
+        let agent = Cohort::init_agent(cfg_agent, cfg_kafka).await;
 
-        cohort.start().await;
+        let database = Database::init_db(cfg_db.clone()).await;
+        let cohort = Cohort::new(agent, Arc::clone(&database));
+
+        prefill_db(database).await;
```

Comment:
Can we seed the database with initial data as a separate binary? This was if we want to test `replicator` or rerun `cohort initiator` without resetting the db everytime.

Reply:
This action is idempotent, can run multiple times without side effect as long as file contains the lower version number. than in DB.

We can think of separating pre-fill into another script.